src/App.js

```diff
@@ -25,17 +25,26 @@ class App extends React.Component {
     state = {
         data: {},
 
+        // (IBN) Can't think of a better name
+        data2: {},
+
     }
 
     async componentDidMount() {
         const fetchedData = await fetchData();
         console.log(fetchedData);
         this.setState({ data: fetchedData })
+
+        // (IBN) Get data for Graph page
+        const fetchedDataForGraph = await graphData();
+        console.log(fetchedDataForGraph);
+        this.setState({ data2: fetchedDataForGraph })
     }
 
     render() {
-        const { data } = this.state;
+        const { data, data2 } = this.state;
         return (
+<<<<<<< HEAD
```

Comment:
Note, this `<<<<<<< HEAD` text is from your merge and you probably want to remove it?